hooks/info

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eu
+
+describe "#B{$(lookup params.installation 'S.H.I.E.L.D.')}"
+    echo
+describe "endpoint information"
+describe "  #C{https://$(lookup params.shield_static_ip)}"
```

Comment:
Recommendation: use exodus url instead of https:// + params.shield_static_ip

Reply:
:+1: